src/VideoPlayer/index.js

```diff
@@ -442,7 +443,10 @@ const videoPlayerPlugin = {
 
 export default autoSetupMixin(videoPlayerPlugin, () => {
   precision =
-    (ApplicationInstance && ApplicationInstance.stage && ApplicationInstance.stage.getRenderPrecision()) || precision
+    (ApplicationInstance &&
+      ApplicationInstance.stage &&
+      ApplicationInstance.stage.getRenderPrecision()) ||
+    precision
```

Comment:
This looks like just linebreaks which has nothing to do with the initial issue, correct?

Reply:
Seems correct.